src/data/javascript-quiz.ts

```diff
@@ -260,6 +260,20 @@ const javascriptQuiz = [
       "Assigning 0 to the length property of the array will clear the contents of the array.",
     Link: "https://www.freecodecamp.org/news/how-javascript-implements-oop/"
   },
+  {
+    Question:
+      "In Javascript class-based OOP, are objects and classes the same constructs?",
+    Answer: "No, objects and classes are two separate constructs.",
+    Distractor1:
+      "No, objects and classes are completely unrelated in JavaScript.",
+    Distractor2:
+      "Yes, objects and classes are only used in functional programming, not in Javascript class-based OOP",
+    Distractor3:
+      "Yes, objects and classes are interchangeable and can be used interchangeably in Javascript class-based OOP.",
+    Explanation:
+      "Classes and objects are two separate constructs, and objects are always created as instances of classes.",
+    Link: "https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_programming"
+  },
```

Comment:
please remove this here because you already included this question in your other PR

Reply:
Thank you for pointing it out, the changes were done.